code/datums/diseases/advance/advance.dm

```diff
@@ -67,40 +76,70 @@
 		),
 		list( //level 5
 			/datum/reagent/drug/space_drugs,
-			/datum/reagent/medicine/mannitol,
-			/datum/reagent/medicine/synaptizine,
 			/datum/reagent/cryptobiolin,
+			/datum/reagent/medicine/coagulant,
+			/datum/reagent/medicine/ammoniated_mercury,
+			/datum/reagent/medicine/neurine,
+			/datum/reagent/toxin/heparin,
 		),
 		list( // level 6
 			/datum/reagent/medicine/antihol,
 			/datum/reagent/medicine/inacusiate,
 			/datum/reagent/medicine/oculine,
-			/datum/reagent/phenol,
+			/datum/reagent/kronkus_extract,
+			/datum/reagent/toxin/chloralhydrate,
+			/datum/reagent/medicine/cryoxadone,
 		),
 		list( // level 7
 			/datum/reagent/medicine/higadrite,
 			/datum/reagent/medicine/leporazine,
 			/datum/reagent/toxin/mindbreaker,
 			/datum/reagent/acetaldehyde,
+			/datum/reagent/acetone_oxide,
 		),
 		list( // level 8
 			/datum/reagent/drug/happiness,
 			/datum/reagent/medicine/ephedrine,
 			/datum/reagent/pax,
+			/datum/reagent/toxin/amatoxin,
+			/datum/reagent/teslium,
+			/datum/reagent/medicine/silibinin,
+			/datum/reagent/bluespace,
 		),
 		list( // level 9
 			/datum/reagent/medicine/sal_acid,
-			/datum/reagent/toxin/chloralhydrate,
+			/datum/reagent/medicine/oxandrolone,
 			/datum/reagent/toxin/lipolicide,
+			/datum/reagent/medicine/earthsblood,
+			/datum/reagent/medicine/atropine,
+			/datum/reagent/toxin/cyanide,
 		),
 		list( // level 10
 			/datum/reagent/drug/aranesp,
 			/datum/reagent/medicine/diphenhydramine,
 			/datum/reagent/pentaerythritol,
+			/datum/reagent/drug/methamphetamine,
+			/datum/reagent/medicine/pyroxadone,
+			/datum/reagent/medicine/salbutamol,
+			/datum/reagent/medicine/pen_acid,
 		),
 		list( //level 11
 			/datum/reagent/medicine/c2/tirimol,
 			/datum/reagent/medicine/modafinil,
+			/datum/reagent/drug/krokodil,
+			/datum/reagent/medicine/omnizine/protozine,
```

Comment:
Another entirely RNG reliant chem that is not at all reasonable go acquire without a considerable amount of legwork. it is easier than kronkus, but this will vary based on whether or not your chemist is on icebox or any other station. Another one in the 'probably too difficult to cure' list